homeassistant/components/keenetic_ndms2/config_flow.py

```diff
@@ -84,10 +87,43 @@ async def async_step_user(self, user_input=None):
             errors=errors,
         )
 
-    async def async_step_import(self, user_input=None):
+    async def async_step_import(self, user_input: dict = None):
         """Import a config entry."""
         return await self.async_step_user(user_input)
 
+    async def async_step_ssdp(self, discovery_info: dict):
+        """Handle a discovered device."""
+        # Filter out items not having "keenetic' in their name
+        if (
+            "keenetic"
+            not in discovery_info.get(ssdp.ATTR_UPNP_FRIENDLY_NAME, "").lower()
+        ):
+            return self.async_abort(reason="not_keenetic_ndms2")
```

Comment:
Abort reason `not_keenetic_ndms2` is missing from the translations file.

Reply:
I don't think it's needed for ssdp discovered case. It wont end up hitting UI.

Reply:
Still, it should be there to be complete.

Reply:
eh...